auslib/test/admin/views/test_history.py

```diff
@@ -96,6 +96,39 @@ def testFieldViewRelease(self):
         self.assertTrue('"fakePartials": true' in ret.data)
         self.assertTrue('"fakePartials": false' in ret.data)
 
+    def testFieldViewDiffFirstRelease(self):
+        # Add first release
+        blob = """
+        {
+            "name": "ddd1",
+            "schema_version": 1,
+            "detailsUrl": "blah",
+            "fakePartials": true,
+            "hashFunction": "sha512",
+            "platforms": {
+                "p": {
+                    "locales": {
+                        "dd": {
+                            "complete": {
+                                "filesize": 1234,
+                                "from": "*",
+                                "hashValue": "abc"
+                            }
+                        }
+                    }
+                }
+            }
+        }"""
+
+        ret = self._put('/releases/ddd1', data=dict(blob=blob, name='ddd1',
+                        product='d', data_version=1))
+        self.assertStatusCode(ret, 201)
+        ret = self._post(
+            '/releases/ddd1',
+            data=dict(data=blob, product='d', data_version=1)
+        )
+        self.assertStatusCode(ret, 200)
+
```

Comment:
This doesn't look like the test you want. The code you've changed is involved with GETs to endpoints like /api/history/diff/release/10189589/data - so you should be making requests like that in your test.